lessons/Data-Science/data-cycle/langs/en-us/pages/notes-data-cycle.adoc

```diff
@@ -27,7 +27,7 @@ Each question a Data Scientist asks adds a chapter to the story of their researc
 
   * *Arithmetic questions* - Answered by doing calculations (comparing, averaging, totaling, etc.) with values from one single column. Examples of arithmetic questions might be “How much does the heaviest animal weigh?” or “What is the average age of animals from the shelter?”
 
-  * *Statistical questions* - These often involve multiple steps to answer, and the answer isn't black and white. When we compare two statistics we are actually comparing two data sets. If we ask "are dogs heavier than cats?", we know that not every dog is heavier than every cat! We just want to know if it is generally true or generally false!
+  * *Statistical questions* - These are questions that both _expect some variability in the data_ related to the question and _account for it in the answers_. Statistical questions often involve multiple steps to answer, and the answers aren't black and white. When we compare two statistics we are actually comparing two data sets. If we ask "are dogs heavier than cats?", we know that not every dog is heavier than every cat! We just want to know if it is generally true or generally false!
```

Comment:
In the last sentence, make "generally" italic and bold in both places. This is a key word that's been enormously helpful in PD, and we should emphasize it more.

Reply:
I tried it that way, but nothing else on the page is bold so it looked strange, so I'm just adding italics.